docs/conceptual/gpu-arch.md

```diff
@@ -5,6 +5,8 @@
   MI100, AMD Instinct">
 </head>
 
+(gpu-arch-documentation)=
```

Comment:
is this necessary? it looks like either it got here by accident, or is missing something on the rhs

Reply:
This is just the weird format that's used with sphinx in markdown for creating references
https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#explicit-targets